x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/EsqlActionTaskIT.java

```diff
@@ -130,19 +156,27 @@ public void testTaskContents() throws Exception {
                 DriverStatus status = (DriverStatus) task.status();
                 assertThat(status.sessionId(), not(emptyOrNullString()));
                 for (DriverStatus.OperatorStatus o : status.activeOperators()) {
+                    logger.info("status {}", o);
                     if (o.operator().startsWith("LuceneSourceOperator[maxPageSize=" + PAGE_SIZE)) {
                         LuceneSourceOperator.Status oStatus = (LuceneSourceOperator.Status) o.status();
-                        assertThat(oStatus.currentLeaf(), lessThanOrEqualTo(oStatus.totalLeaves()));
-                        assertThat(oStatus.slicePosition(), greaterThanOrEqualTo(0));
-                        if (oStatus.sliceSize() != 0) {
-                            assertThat(oStatus.slicePosition(), lessThanOrEqualTo(oStatus.sliceSize()));
+                        assertThat(oStatus.processedSlices(), lessThanOrEqualTo(oStatus.totalSlices()));
+                        assertThat(oStatus.sliceIndex(), lessThanOrEqualTo(oStatus.totalSlices()));
+                        assertThat(oStatus.sliceMin(), greaterThanOrEqualTo(0));
+                        assertThat(oStatus.sliceMax(), greaterThanOrEqualTo(oStatus.sliceMin()));
+                        if (oStatus.sliceMin() != 0 && oStatus.sliceMin() != 0) {
```

Comment:
nit: sliceMin -> sliceMax for the second clause.

Reply:
Ooof. Got i.